core/reporter-impl/src/test/java/org/arquillian/reporter/impl/asserts/ReportAssert.java

```diff
@@ -151,6 +149,35 @@ public REPORTASSERTTYPE hasEntriesContaining(List<Entry> expectedEntries) {
         return (REPORTASSERTTYPE) this;
     }
 
+    public REPORTASSERTTYPE hasEntryContainingKeys(StringKey... expectedStringKey) {
+        return hasEntryContainingKeys(Arrays.asList(expectedStringKey));
+    }
+
+    public REPORTASSERTTYPE hasEntryContainingKeys(List<StringKey> expectedStringKeys) {
+        isNotNull();
+
+        expectedStringKeys.parallelStream().forEach(expectedStringKey -> {
```

Comment:
I don't think that it is necessary to have a parallel stream there - it is not performed on huge data, is it? The reason why I would avoid using the parallel stream in assertions is the fact that when the test fails then the exception/message doesn't look very intuitive. I'm using the parallel stream in some assertions because those ones are used in big loops on huge data sets  - without the parallel stream, execution of some tests would take several minutes longer - but this is not the case.